package.json

```diff
@@ -15,18 +15,15 @@
     "url": "https://github.com/opstalent/React-Redux-Startekit"
   },
   "scripts": {
-    "start": "webpack-dev-server --inline --hot --history-api-fallback --host localhost --port 8080",
-    "build-dev": "webpack --config webpack.dev.config.js",
-    "build-prod": "webpack -p --config webpack.prod.config.js",
-    "test": "mocha --recursive --compilers js:babel-register --require babel-polyfill --require ignore-styles",
-    "test-watch": "npm test -- --watch",
+    "start": "webpack-dev-server --config webpack.dev.config.js --history-api-fallback --host localhost --port 8080",
+    "build": "webpack -p --config webpack.prod.config.js",
+    "test": "jest ./test",
+    "test:watch": "npm test -- --watch",
     "lint": "npm run lint:js",
     "lint:eslint": "eslint --ignore-path .gitignore --ignore-pattern internals/scripts",
     "lint:js": "npm run lint:eslint -- . ",
     "lint:fix": "eslint --fix --ignore-path .gitignore --ignore-pattern internals/scripts -- .",
     "lint:staged": "lint-staged",
```

Comment:
why there is so many lint scripts?

Reply:
I don't know, I haven't been changing them ;)